Nodejs/Setup/BuildRelease.ps1

```diff
@@ -553,18 +565,35 @@ try {
                     $i.signed_bindir $approvers `
                     $project_name $project_url "$project_name $($i.VSName) - native code" $project_keywords `
                     "authenticode" 
+
+                # we only loc Dev 15
+                if ($i.VSTarget -eq "15.0") {
+                    foreach ($loc in $locales) {
+                        $jobs += begin_sign_files `
+                            @($localized_files | %{@{path="$($i.unsigned_bindir)\$loc\$_"; name=$_}} | ?{Test-Path $_.path}) `
+                            "$($i.signed_bindir)\$loc" $approvers `
+                            $project_name $project_url "$project_name $($i.VSName) - managed code - $loc" $project_keywords `
+                            "authenticode;strongname" `
+                            -delaysigned
+                    }
+                }
             }
             
             end_sign_files $jobs
             
             foreach ($i in $target_info) {
+                Write-Output "Begin symbol submission for $($i.VSName)"
                 if ($i.logfile -in $failed_logs) {
                     Write-Output "Skipping symbol submission for $($i.VSName) because the build failed"
                     continue
                 }
                 submit_symbols "$project_name$spacename" "$buildnumber $($i.VSName) $config" "binaries" $i.signed_bindir $symbol_contacts
                 submit_symbols "$project_name$spacename" "$buildnumber $($i.VSName) $config" "symbols" $i.symboldir $symbol_contacts
 
+                Write-Output "End symbol submission for $($i.VSName)"                
+            }
+
+            foreach ($i in $target_info) {
```

Comment:
Why did you separate this out into another iteration over $target_info? The extra loop doesn't check failed logs and skip the setup build if the build failed. Is that correct?